packages/grafana-ui/src/components/index.ts

```diff
@@ -14,6 +14,8 @@ export { IndicatorsContainer } from './Select/IndicatorsContainer';
 export { NoOptionsMessage } from './Select/NoOptionsMessage';
 export { default as resetSelectStyles } from './Select/resetSelectStyles';
 export { ButtonSelect } from './Select/ButtonSelect';
+export { ToggleButtonGroup, ToggleButton } from './ToggleButtonGroup/ToggleButtonGroup';
+export * from './ToggleButtonGroup/types';
```

Comment:
Why are you exposing the types from grafana/ui?

Reply:
As the state is being passed into `onChange`. Could always just pass on just `selected` and `value` and just forget about the type alltogether.

https://github.com/grafana/grafana/pull/17871/files/fd7a43782e6dc117f2f4b0f26bc2af6a16391d14#diff-41a0c07d4ace9d7b762563c546be3f89R116